src/relaycard/card.py

```diff
@@ -1,38 +1,47 @@
 import logging
-import serial
 import time
 
-from .constants import (CMD_SETUP, CMD_GETPORT, CMD_SETPORT, CMD_TOGGLE,
-    CMD_SETSINGLE, CMD_DELSINGLE, RESP_GETPORT, RESP_SETPORT, RESP_TOGGLE,
-    RESP_SETSINGLE, RESP_DELSINGLE)
-from .frame import ResponseFrame, RequestFrame
-from .state import RelayState
+import serial
 
+from .constants import (
+    CMD_DELSINGLE,
+    CMD_GETPORT,
+    CMD_SETPORT,
+    CMD_SETSINGLE,
+    CMD_SETUP,
+    CMD_TOGGLE,
+    RESP_DELSINGLE,
+    RESP_GETPORT,
+    RESP_SETPORT,
+    RESP_SETSINGLE,
+    RESP_TOGGLE,
+)
+from .frame import RequestFrame, ResponseFrame
+from .state import RelayState
 
-class RelayCard(object):
-    port = None
```

Comment:
Is there a reason why you removed the port and card_count initial values? Without these, the `is_initialized` property would explode if called before `setup`.